packages/galata/src/galata.ts

```diff
@@ -1251,7 +1251,7 @@ export namespace galata {
     }
 
     export async function getToolbarItem(
-      itemId: NotebookToolbarItemId,
+      itemId: NotebookToolbarItemId | string,
```

Comment:
Wondering whether there could be new types for both the notebook toolbar items and sidebar tabs, and have the `| string` in it.

For example:

```typescript
// the list of default items in JupyterLab
export type DefaultNotebookToolbarItemId = 'save' | 'insert' ....;
export type NotebookToolbarItemId = DefaultNotebookToolbarItemId | string;
```

And something similar for the sidebar tab ids.

Although this is more for readability and wouldn't change the current behavior.

Reply:
This is a nice suggestion. I'll update the PR.

Reply:
:heavy_check_mark: